### DISCUSSION TITLE
Game closed itself while bad internet connection on playlists

### DISCUSSION THREAD

**Discussion by TheRealStevie** [ppy/osu#20448] [Q&A] [answered]
While I had bad connection on playlist result screen and then trying to go back out, the game just closed itself without displaying any dialouge.
[database.log](https://github.com/ppy/osu/files/9638426/database.log)
[network.log](https://github.com/ppy/osu/files/9638427/network.log)
[performance.log](https://github.com/ppy/osu/files/9638428/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9638429/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9638430/updater.log)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/pull/20504
